src/draw/DefaultDrawer.js

```diff
@@ -341,6 +344,46 @@ class DefaultDrawer {
     }
   }
 
+  /**
+   * Delete a link and his component attribute.
+   * @param {ComponentLink[]} links - Array that contains all links.
+   * @private
+   */
+  __deleteLinks(links) {
+    this.d3.selectAll('.link').on('contextmenu', (event) => {
+      event.preventDefault();
+      const currentLink = this.d3.select(event.currentTarget).datum();
+      const { target, source } = currentLink;
+
+      links.forEach((link, index) => {
+        if (link === currentLink) {
+          links.splice(index, 1);
+          this.drawLinks(links);
+        }
+      });
+      const sourceComponent = this.d3.select(`#${source}`).datum();
+
+      const currentAttribute = sourceComponent.attributes
+        .find((attr) => attr.definition.type === 'Link'
+          && attr.value.includes(target)
+          && attr.name === currentLink.definition.attributeRef);
+
+      currentAttribute.value.forEach((item, index) => {
+        if (item === target) {
+          currentAttribute.value.splice(index, 1);
+        }
+      });
+
+      if (currentAttribute.value.length === 0) {
```

Comment:
```js
sourceComponent.attributes = sourceComponent.attributes.filter((c) => !(c.definition.type === 'Link' && c.definition.name === attributeRef && c.value === []))
```

Reply:
That filter will never work since [] !== [],
simply use 
```js
const index = sourceComponent.attributes.findIndex(a => a === currentAttribute)
if (index >=0)
    sourceComponent.attributes.splice(index, 1);
```